daprdocs/content/en/reference/components-reference/supported-pubsub/setup-azure-servicebus-queues.md

```diff
@@ -69,26 +68,25 @@ The above example uses secrets as plain strings. It is recommended to use a secr
 |--------------------|:--------:|---------|---------|
```

Comment:
Looking at the docs here https://docs.dapr.io/reference/components-reference/supported-pubsub/setup-azure-servicebus/ the third column "Details" is narrow in width and makes this page very long to scroll down. Could you make this column wider and "steal" from the other columns in width?

Reply:
I can't control the width of the columns with Markdown. But I changed the example to "see above"